### PR TITLE
Video

### PR DESCRIPTION
**Pull request by lc512k**
cc @commuterjoy 
gh issue: https://github.com/Financial-Times/spoor-enrichment/issues/311

### CODE REVIEW

File: server/models/event.js [hunk 1]
```diff
@@ -27,7 +27,7 @@ EventModel.prototype._sqsToJson = function () {
 				`api -> sns -> sqs -> enrichment -> pipeline`
 
 			... so here we need to unpack the SQS message
-			from it's SNS envelope.
+			from its SNS envelope.
 		*/
```
Comment (adambraimbridge) [inline server/models/event.js]: 💯

File: server/transforms/video.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+		if (!contentId) {
+			logger.info('transforms/video no contentId');
+			return Promise.resolve({});
+		}
+
+		logger.info(`transforms/video fetching ${this.url} ${contentId}`);
+
+		return this.fetch(host + contentId, {
+			timeout: 2000
```
Comment (commuterjoy) [inline server/transforms/video.js]: are you going to transform the results?

File: server/transforms/video.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+const err = e => logger.error('transforms/video', e);
+
+const host = 'https://next-video.ft.com/api/';
+
+class Video extends Transform {
+
+	run () {
+
+		const contentId = this.event.pluck('context.contentId');
```
Comment (commuterjoy) [inline server/transforms/video.js]: Normally, when I'm testing out the load on the APIs etc. I would scope this to a smaller number of events until I understand the throughput and it's effect on the API.

`this.event.pluck('category') === 'video' && this.event.pluck('action') === 'play'`

**Comment by commuterjoy** [comment]
+1 good - aside from comments

File: server/transforms/video.js (hunk 2 above)
Comment (lc512k) [inline server/transforms/video.js]: We don't really need to, do we? Save for perhaps removing things we don't need (yet) from the object that's returned by the API? Are there performance/storage implications if we decorate the event whole object, as ^^?

Comment (commuterjoy) [inline server/transforms/video.js]: No storage implications really, just I tend to weed out spurious stuff. Lets merge without and we can change later.

**Pull request merged**